VsViewer/vs_calc/VsProfile.py

```diff
@@ -158,7 +217,10 @@ def to_dataframe(self):
         """
         Turns the VsProfile into a dataframe for download
         """
-        return pd.DataFrame(np.column_stack([self.depth, self.vs, self.vs_sd]), columns=["Depth", "Vs", "Vs_SD"])
+        return pd.DataFrame(
+            np.column_stack([self.depth, self.vs, self.vs_sd]),
+            columns=["Depth", "Vs", "Vs_SD"],
+        )
```

Comment:
I assume this is @lee-robin's code, but you could avoid the column stack here and use a dictionary to construct the dataframe instead.